### DISCUSSION TITLE
Where is `copy_required_dlls()` coming from?

### DISCUSSION THREAD

**Discussion by markusbkk** [DiligentGraphics/DiligentEngine#321] [General]
When I try to build a "HelloDiligent" project as outlined [here](https://github.com/DiligentGraphics/DiligentEngine?tab=readme-ov-file#your-project-uses-cmake) CMake errors out with `Unknown CMake command "copy_required_dlls".`

Where is this function coming from? Is this a CMake built-in or something created by Diligent? Is it only available for Windows (I'm on GNU/Linux)?

**Comment by TheMostDiligent** [comment]
It is a utility function [defined here](https://github.com/DiligentGraphics/DiligentCore/blob/master/BuildTools/CMake/BuildUtils.cmake#L3). It is available on Windows only and is used to copy the dlls next to the executable.
